core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -297,6 +297,9 @@
             exceptionLoggingMode("annotatedServiceExceptionVerbosity",
                                  DEFAULT_ANNOTATED_SERVICE_EXCEPTION_VERBOSITY);
 
+    private static final boolean DEFAULT_USE_EVENT_LOOP_FOR_ANNOTATED_SERVICE =
+            getBoolean("defaultUseEventLoopForAnnotatedService", false);
```

Comment:
How about not introducing any flag for this new behavior and letting our users just use `@Blocking`, so that we do not have to maintain this way too long flag?